pkg/solidity-utils/test/foundry/LogExpMath.t.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.7.0;
+
+import "forge-std/Test.sol";
+
+import "../../contracts/math/LogExpMath.sol";
+import "../../contracts/math/ReferenceLogExpMath.sol";
+
+contract LogExpMathsTest is Test {
+    function testExpEquivalence(int256 a) external {
+        vm.assume(a > ReferenceLogExpMath.MIN_NATURAL_EXPONENT);
+        vm.assume(a < ReferenceLogExpMath.MAX_NATURAL_EXPONENT);
+
+        int256 expectedExp = ReferenceLogExpMath.exp(a);
+        int256 actualExp = LogExpMath.exp(a);
+
+        assertApproxEqRel(actualExp, expectedExp, 1e2);
```

Comment:
It's good to test that they are quite close, but we also definitely want some ffi testing to test that the maximum error remains what it used to be.